### ISSUE TITLE
 Feature Request: favicon.ico

### ISSUE THREAD

**Issue by rndm2** [MatMoul/quickdial-webext#43]
Looks better with favicon ico. 

PS really need custom css

**Comment by rndm2** [comment]
I can make changes to support custom css and fix some issues. 
But I need to know you will accept my PR

**Comment by MatMoul** [comment]
favicon will comming in future release but I have other priorities.

PR are accepted but for custom css, It's possible that the structure of the dial page change...
If you a make a PR for this, I suggest to add a tab on Quick Dial settings page with a caution message that is only for power user...

**Comment by fernandosantucci** [comment]
I agree that favicons are implemented as long as there is an option to turn them off on the cells tab in settings. This has always been one of the things that upset my grid layout in Fast Dial.

**Comment by MatMoul** [comment]
@fernandosantucci Don't worry, a setting will be provided

**Comment by XedMada** [comment]
A feature i'd _really_ love to have, as 80% of the "custom" icons i've created were pulled directly off the website's "apple-touch-icon".  Since it isn't specifically stated here, i'd like to note that the literal favicon.ico should be used as a last resort, preferring the now ubiquitous "apple-touch-icon" and Web App Manifest's icon definition.

For completeness sake I'll also note "mask-icon" should be considered. If it were added one would expect to be capable of manually changing the color, which would probably make the whole thing a separate feature.

**Comment by MatMoul** [comment]
@XedMada I'm not sure I understand, can you give some screenshot of what you want

**Comment by XedMada** [comment]
Most professional websites these days provide high quality icons as part of their [Web App Manifest](https://www.w3.org/TR/appmanifest/#icons-member). Basically it is the favicon replacement.

I'd love the ability to set a bookmark's picture to be this automatically provided image instead of the generated screen capture or a locally defined image (which in my case is almost always the site icon I downloaded from its manifest.)
